modules/core/shared/src/main/scala/lucuma/core/enum/GnirsPrism.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import lucuma.core.util.Enumerated
+
+/**
+ * Enumerated type for GNIRS Prism Turret.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GnirsPrism(
+  val tag: String,
+  val shortName: String,
+  val longName: String
+) extends Product with Serializable {
+  type Self = this.type
+}
+
+object GnirsPrism {
+
+  type Aux[A] = GnirsPrism { type Self = A }
```

Comment:
Remind me what this is about?

Reply:
Not needed. It was from that thing where I was experimenting with indexing thpes on the singleton types of enum values, but it was too complicated.